src/main/java/com/feelfreecincinncati/service/iAdventureService.java

```diff
@@ -4,8 +4,8 @@
 
 public interface iAdventureService {
 
-	AdventureDTO fetchById(int id);
+	public AdventureDTO fetchById(int id);

```

Comment:
Access modifiers generally aren't needed on interfaces, unless there is a specific reason.